src/utils/browserUtil.ts

```diff
@@ -0,0 +1,9 @@
+import * as openpackage from 'open';
+
+export const browserUtil = {
+  /* c8 ignore next 4 */
```

Comment:
Is there a reason why we're excluding the whole method from testing?

Reply:
I could not find a way on how to stub the `open` package, which, I think, is needed to test the `{wait: true}`. I also asked @milanholemans and he told me that it would be fine to exclude this 'as this wasn't rocket science'

Reply:
It's quite annoying to test because we cannot stub an npm package. There are some tricks we could do like introducing another property in the util which points to the `open` package. But that would mean that this would also be exposed in the util, which is far from ideal and would be very confusing.
Therefore I'd just let it ignore by c8. According to me the logic in this function is so small that it wouldn't really harm.

But if anyone has a clean way to test the behavior, we should definitely do it. I just couldn't come up with one.